azure_functions_worker/bindings/meta.py

```diff
@@ -184,3 +216,43 @@ def to_outgoing_param_binding(binding: str, obj: typing.Any, *,
         return protos.ParameterBinding(
             name=out_name,
             data=rpc_val)
+
+
+def get_deferred_binding(bind_name: str,
+                         pytype: typing.Optional[type] = None) -> object:
+    binding = None
+
+    # Checks if pytype is a supported sdk type
+    if (SDK_BINDING_REGISTRY is not None
+            and SDK_BINDING_REGISTRY.check_supported_type(pytype)):
+        # Set flag once
+        global deferred_bindings_enabled
+        if not deferred_bindings_enabled:
+            deferred_bindings_enabled = True
+        binding = SDK_BINDING_REGISTRY.get(bind_name)
+
+    # This will return None if not a supported type
+    return binding
+
+
+def deferred_bindings_decode(binding: typing.Any,
+                             pb: protos.ParameterBinding, *,
+                             pytype: typing.Optional[type],
+                             datum: typing.Any,
+                             metadata: typing.Any):
+    global SDK_CACHE
+
+    if SDK_CACHE is None:
```

Comment:
the cache is a dictionary. It will never be none. check if there are any keys in the dictionary

Reply:
I call get() and provide a default value. If the dict is empty or the key doesn't exist, obj will be None as expected